GNUmakefile

```diff
@@ -3,10 +3,17 @@ PROJECT_ROOT := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 THIS_OS := $(shell uname | cut -d- -f1)
 THIS_ARCH := $(shell uname -m)
 
+REPO = github.com/hashicorp/nomad
```

Comment:
will ent need a different value?

Reply:
that crossed my mind, but nope!  actually though since we both bumped into it, I'll rename it -- it's not actually the repo, it's the go module, so `GO_MODULE` is more accurate (and I don't think conflicts with any other go env names).